pkgs/development/libraries/stb/default.nix

```diff
@@ -14,8 +14,17 @@ stdenv.mkDerivation {
   dontBuild = true;
 
   installPhase = ''
-    mkdir -p $out/include/stb
+    mkdir -p $out/include/stb $out/lib/pkgconfig
     cp *.h $out/include/stb/
+
+    cat << EOF > $out/lib/pkgconfig/stb.pc
+    prefix=''${out}
+    includedir=\''${prefix}/include/stb
+    Cflags: -I\''${includedir}
```

Comment:
The example in this [pkg-config guide](https://people.freedesktop.org/~dbn/pkg-config-guide.html) would have us write:

```
includedir=\''${prefix}/include
Cflags: -I\''${includedir}/stb
```

What do you think?